erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.py

```diff
@@ -66,8 +66,7 @@ def on_submit(self):
 				parent=%s""", (d.sales_person, d.item_code, self.name), as_dict=1)
 
 			for key in scheduled_date:
-				description = frappe._("Reference: %s, Item Code: %s and Customer: %s") % \
-					(self.name, d.item_code, self.customer)
+				description = frappe._("Reference: %s, Item Code: %s and Customer: %s").format(self.name, d.item_code, self.customer)
```

Comment:
frappe._("Reference: %s, Item Code: %s and Customer: %s").format(self.name, d.item_code, self.customer) should be frappe._("Reference: {0}, Item Code: {1} and Customer: {2}").format(self.name, d.item_code, self.customer)